api-lambdas/lambdas/get_lambda.py

```diff
@@ -0,0 +1,13 @@
+from scipy.optimize import curve_fit
+import numpy as np
+
+
+def ley_potencia(T, Lambda, No):
```

Comment:
Por favor usa el mismo idioma al nombrar las funciones a lo largo de este archivo
```suggestion
def power_law(T, Lambda, No):
```